src/Microsoft.ServiceHub.Framework/ServiceBrokerClient.cs

```diff
@@ -53,7 +53,7 @@ public class ServiceBrokerClient : IDisposableObservable
 	/// <summary>
 	/// Initializes a new instance of the <see cref="ServiceBrokerClient"/> class.
 	/// </summary>
-	/// <param name="serviceBroker">The underlying service broker.</param>
+	/// <param name="serviceBroker">The underlying service broker. This will be disposed of if it implements <see cref="IDisposable" /> when this <see cref="ServiceBrokerClient" /> is disposed of.</param>
```

Comment:
Should we have an "ownsServiceBroker" overload then to avoid disposing the underlying broker in some cases? Or would the caller just be expected to pass a wrapped broker that doesn't implement IDisposable in those cases?

Reply:
At the moment, yes I'm telling people to use `ServiceBrokerAggregator.NonDisposable` to retain ownership.